pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -166,6 +167,10 @@ func Run() {
 
 	triggerDeviceInfo := make(chan struct{}, 1)
 	zedagentCtx := zedagentContext{TriggerDeviceInfo: triggerDeviceInfo}
+	zedagentCtx.globalConfigPtr = &types.GlobalConfig{}
+	*zedagentCtx.globalConfigPtr = types.GlobalConfigDefaults
+	zedagentCtx.globalConfigPtr.Errors = make([]string, 0)
```

Comment:
This is odd. Any errors should be in a globalStatusPtr and not in the config.
Since the GlobalConfig is a persistent thing, your current code is presumably persisting any error string as well. That isn't what we want, is it?

Reply:
Hmm.. I agree.. Having a GlobalStatus is better.. Right now, it will only have Errors as the field.
Will make the change..